samples/angular/src/app/components/profile-form/profile-form.component.html

```diff
@@ -0,0 +1,72 @@
+<div *ngIf="form">
+  <div class="form" [formGroup]="form">
+    <div class="form-input">
+      <label for="name">Name *</label>
+      <ef-text-field
+      id="name"
+      class="form-field"
+      formControlName="name"
+      [error]="name && name.invalid && (name.dirty || name.touched)"
+      ngDefaultControl></ef-text-field>
+    </div>
+    <div class="form-input">
+      <label id="gender_group">Gender</label>
+      <div role="radiogroup" aria-labelledby="gender_group" #genderGroup>
+        <ef-radio-button name="gender" value="male" [checked]="gender === 'male'? 'true': undefined">Male</ef-radio-button>
```

Comment:
Do we need to bind back to Angular? I think checked property do correctly without this.

Reply:
Agree, I will remove it.